### ISSUE TITLE
no-unused-vars error false positive

### ISSUE THREAD

**Issue by yarom82** [mightyiam/eslint-config-standard-with-typescript#5]
The error is in line 2 - VNode
``` ts
import { test } from 'ava'
import { VNode } from 'snabbdom/vnode'
import { ul } from '@cycle/dom'
import subject from '.'

test('empty content', (t) => {
  const input: VNode[] = []
  const expected = ul()
  const actual = subject(input)
  t.deepEqual(actual, expected)
})
```

**Comment by mightyiam** [comment]
Thank you, @yarom82. I've opened an issue upstream:
https://github.com/nzakas/eslint-plugin-typescript/pull/77

**Comment by mightyiam** [comment]
Closing with #6.
